<launch>
	<node name="manip_controller" pkg="manip_control" type="manip_controller.py" output="screen">
		<rosparam file="$(find manip_control)/config/kinematics.yaml" command="load"/>
		<rosparam file="$(find manip_control)/config/controller.yaml" command="load"/>
	</node>
</launch>
